src/main/java/pokecube/legends/init/ItemInit.java

```diff
@@ -141,7 +141,7 @@
         PWATER = PokecubeLegends.ITEMS.register("pwater", () -> new ItemBase("pwater", 1).setTooltipName("pwater"));
 
         WISHING_PIECE = PokecubeLegends.ITEMS.register("wishing_piece", () -> new ItemBase("wishing_piece", 1));
-        GIGANTIC_SHARD = PokecubeLegends.ITEMS.register("gigantic_shard", () -> new GiganticShard("gigantic_shard", 1));
+        //GIGANTIC_SHARD = PokecubeLegends.ITEMS.register("gigantic_shard", () -> new GiganticShard("gigantic_shard", 1));
```

Comment:
Do not remove the item after the update was already released to add it, this will cause incompatiblity issues... just mark in the name that it isn't working